packages/features/shell/Shell.tsx

```diff
@@ -787,7 +787,11 @@ export function ShellMain(props: LayoutProps) {
   return (
     <>
       <div
-        className={classNames("mb-6 flex items-center sm:mt-0", props.smallHeading ? "lg:mb-7" : "lg:mb-10")}>
+        className={classNames(
+          "mb-6 flex items-center sm:mt-0",
+          props.smallHeading ? "lg:mb-7" : "lg:mb-10",
```

Comment:
@PeerRich @Jaibles 
Noticed this recent change adds a different margin bottom than required by the ticket.
What should be the bottom margin? `32px` or `40px`?

![Screenshot 2023-01-30 at 6 57 56 PM](https://user-images.githubusercontent.com/121884634/215486945-ca5387c2-2199-4ddd-95b8-7d72a4ac9b39.png)